src/main/java/org/dependencytrack/parser/snyk/SnykParser.java

```diff
@@ -204,45 +208,61 @@ public List<VulnerableSoftware> parseVersionRanges(final QueryManager qm, final
+                        versionStartIncluding = null;
+                    }
+                    if (versionEndIncluding.length() == 0 || versionEndIncluding.contains("*")) {
+                        versionStartIncluding = null;
+                    }
+                } else { //since we are not able to parse specific range, we do not want to end up with false positives and therefore this part will be skipped from being saved to db.
+                    LOGGER.warn("Range not definite. Not saving this vulnerable software information. The purl was: "+purl);
                 }
             }
-
-            VulnerableSoftware vs = qm.getVulnerableSoftwareByPurl(packageURL.getType(), packageURL.getNamespace(), packageURL.getName(),
-                    versionEndExcluding, versionEndIncluding, versionStartExcluding, versionStartIncluding);
-            if (vs == null) {
-                vs = new VulnerableSoftware();
-                vs.setVulnerable(true);
-                vs.setPurlType(packageURL.getType());
-                vs.setPurlNamespace(packageURL.getNamespace());
-                vs.setPurlName(packageURL.getName());
-                vs.setVersion(packageURL.getVersion());
-                vs.setVersionStartIncluding(versionStartIncluding);
-                vs.setVersionStartExcluding(versionStartExcluding);
-                vs.setVersionEndIncluding(versionEndIncluding);
-                vs.setVersionEndExcluding(versionEndExcluding);
+            //check for a numeric definite version range
+            if ((versionStartIncluding != null && versionEndIncluding != null) || (versionStartIncluding != null && versionEndExcluding != null) || (versionStartExcluding != null && versionEndIncluding != null) || (versionStartExcluding != null && versionEndExcluding != null)) {
+                VulnerableSoftware vs = qm.getVulnerableSoftwareByPurl(packageURL.getType(), packageURL.getNamespace(), packageURL.getName(), versionEndExcluding, versionEndIncluding, versionStartExcluding, versionStartIncluding);
+                if (vs == null) {
+                    vs = new VulnerableSoftware();
+                    vs.setVulnerable(true);
+                    vs.setPurlType(packageURL.getType());
+                    vs.setPurlNamespace(packageURL.getNamespace());
+                    vs.setPurlName(packageURL.getName());
+                    vs.setVersion(packageURL.getVersion());
+                    vs.setVersionStartIncluding(versionStartIncluding);
+                    vs.setVersionStartExcluding(versionStartExcluding);
+                    vs.setVersionEndIncluding(versionEndIncluding);
+                    vs.setVersionEndExcluding(versionEndExcluding);
+                }
+                vulnerableSoftwares.add(vs);
+            } else {
+                LOGGER.warn("Range not definite. Not saving this vulnerable software information. The purl was: "+purl);
```

Comment:
Please lower this to `debug`, as there is nothing users can do against it, and it produces a lot of noise.

Reply:
done